README.md

```diff
@@ -6,5 +6,86 @@ This is an Android SDK for the [KUKSA Vehicle Abstraction Layer](https://github.
 [![License](https://img.shields.io/badge/License-Apache%202.0-green.svg)](https://opensource.org/licenses/Apache-2.0)
 [![Gitter](https://img.shields.io/gitter/room/kuksa-val/community)](https://gitter.im/kuksa-val/community)
 
+## Overview
```

Comment:
All code snippets are kotlin only. Since we want to be 100% compatible with Java code, please also add code snippets for Java. 

A toggle would be optimal, if this is not feasable maybe add a separate headline Kotlin / Java in the Quickstart guide